### PR TITLE
Update changelog and docs for the new platforms

### PR DESCRIPTION
**Pull request by saolsen**

### CODE REVIEW

File: docs/changelogs/vNEXT.rst [hunk 1]
```diff
@@ -24,6 +24,14 @@ Link to migration guide
 New features
 ==============
 
+Windows Support
+===============
+The three oso libraries (python, ruby and java) all now work on Windows.
```
Comment (dhatch) [inline docs/changelogs/vNEXT.rst:29]:
```suggestion
The three oso libraries (Python, Ruby and Java) all now work on Windows.
```

File: docs/changelogs/vNEXT.rst [hunk 2]
```diff
@@ -24,6 +24,14 @@ Link to migration guide
...
+
+Musl Python Builds
+===========
+Python linux musl-based wheels (for alpine and other musl based distros) are built and downloadable from the release in Github.
```
Comment (dhatch) [inline docs/changelogs/vNEXT.rst:33]: Might be nice to add a link here to the releases page.

Comment (gj) [inline docs/changelogs/vNEXT.rst:33]:
```suggestion
musl builds for Python
======================

musl-based Python wheels (for Alpine Linux and other musl-based distros) are built and downloadable from [the releases page on GitHub](https://github.com/osohq/oso/releases).
```

Comment (gj) [reply docs/changelogs/vNEXT.rst:33]: ah crap, that's a markdown-style link

Comment (samscott89) [reply docs/changelogs/vNEXT.rst:33]: We're going to publish it as `oso-alpine` right?

Comment (samscott89) [reply docs/changelogs/vNEXT.rst:33]: Or `oso-musl` I forget.

Comment (gj) [reply docs/changelogs/vNEXT.rst:33]: updated (I think)

**Comment by dhatch** [comment]
I don't think we made a decision on how to publish it

File: docs/changelogs/vNEXT.rst (hunk 2 above)
Comment (saolsen) [reply docs/changelogs/vNEXT.rst:33]: It's not published to pypi at all yet. It's just available as a download on the Github release.

**Comment by saolsen** [comment]
We never decided to publish it.

File: docs/download.rst [hunk 3]
```diff
@@ -14,6 +14,9 @@ oso is available as a library in several languages:
 
             $ pip install oso=={release}
 
+        The python version is known to work on glibc based distributions but not on musl based ones (like apline).
+        Wheels built against musl that you can use on alpine can be downloaded from the `Release <https://github.com/osohq/oso/releases/latest>`_ on Github.
```
Comment (gj) [inline docs/download.rst:18]:
```suggestion
        The Python version is known to work on glibc-based distributions but not on musl-based ones (like Alpine Linux).
        Wheels built against musl that you can use on Alpine Linux can be downloaded from `the releases page on GitHub <https://github.com/osohq/oso/releases/latest>`_.
```

File: docs/changelogs/vNEXT.rst [hunk 4]
```diff
@@ -24,6 +24,15 @@ Link to migration guide
...
 ==============
 
+Windows Support
+===============
+The three oso libraries (Python, Ruby and Java) all now work on Windows.
+
+musl builds for Python
+======================
+
+musl-based Python wheels (for Alpine Linux and other musl-based distros) are built and downloadable from `the releases page on GitHub <https://github.com/osohq/oso/releases>`_.
```
Comment (gj) [inline docs/changelogs/vNEXT.rst:34]:
```suggestion
musl-based Python wheels (for Alpine Linux and other musl-based distros) are built and downloadable from `the releases page on GitHub <https://github.com/osohq/oso/releases/latest>`_.
```

File: docs/getting-started/quickstart.rst [hunk 5]
```diff
@@ -78,10 +78,11 @@ Adding oso
...
 
-      Go to the `Maven Repository <https://github.com/osohq/oso/packages/321403>`_
-      and download the latest jar.
+      Go to the `Maven Repository <https://search.maven.org/artifact/com.osohq/oso>`_
 
-      Either add this to your Java project libraries,
+      Either download the latest jar and add this to your Java project libraries,
+      load using your IDE,
+      add the project as dependency to your build system
       load using your IDE,
```
Comment (gj) [inline docs/getting-started/quickstart.rst:86]: looks like this line is now repeated
```suggestion
```

**Pull request merged**